projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -197,13 +192,8 @@ release-ami-ubuntu-2004: setup-ami-share deps-ami setup-packer-configs-ami
 	build/export-ami-to-s3.sh $(RELEASE_BRANCH) $(MANIFEST_OUTPUT) raw $(EXPORT_AMI_DST) $(LATEST_AMI_S3_URL)
 
 .PHONY: release-ova-%
-release-ova-%: MAKEFLAGS=
-release-ova-%: FULL_OUTPUT_DIR=$(MAKE_ROOT)/$(OUTPUT_DIR)/ova/$(IMAGE_OS)
-release-ova-%: PACKER_TYPE_VAR_FILES=$(PACKER_OVA_VAR_FILES)
-release-ova-%: deps-ova setup-vsphere setup-packer-configs-ova
-	PACKER_FLAGS="-force" PACKER_LOG=1 PACKER_LOG_PATH=$(ARTIFACTS_PATH)/ova/$(IMAGE_OS)/packer.log PACKER_VAR_FILES="$(PACKER_VAR_FILES)" \
-		OVF_CUSTOM_PROPERTIES="$(FULL_OUTPUT_DIR)/config/ovf_custom_properties.json" \
-		$(MAKE) -C $(IMAGE_BUILDER_DIR) build-node-ova-vsphere-$*
+release-ova-%:
+	build/build_ova_image_cli.sh $* $(RELEASE_BRANCH)
 
```

Comment:
Here the stem to the make target is something like `ubuntu-2004` or `rhel-8` that directly maps to the upstream image builder target, `build-node-ova-vsphere-ubuntu-2004` etc. If using this as Image OS, we will need to trim it.

Reply:
I changed the stem above to $(IMAGE_OS). Yea we were mapping them to upstream targets, but since we are transitioning to use the cli, the idea is to get rid of it and let cli control that either with input args/custom base image.

For now the cli dictates the version of the OS that we of course validate. In the near future this could be an input argument to the cli or a custom base image itself could serve as an input. Because of these, I got rid of version here.

If you feel otherwise, we can talk about it.